pkg/version/version.go

```diff
@@ -62,8 +63,8 @@ type rosaVersion struct {
 	retriever Retriever
 }
 
-func (v rosaVersion) IsLatest(latestVersion string) (*goVer.Version, bool, error) {
-	currentVersion, err := goVer.NewVersion(latestVersion)
+func (v rosaVersion) IsLatest(version string) (*goVer.Version, bool, error) {
```

Comment:
This is an opinionated change more than a functional one - the variable name `latestVersion` confused me for a bit here since it's not actually the latest version, it's what we're checking to _see_ if is the latest version.